composedb.config.json

```diff
@@ -1,23 +1 @@
-{
-  "anchor": {},
-  "http-api": {
-    "cors-allowed-origins": [".*"],
-    "admin-dids": ["did:key:z6MkfuigJH7DXP6PiYpMVLvsX1Ak2CYKmMEX488zCub9tvwP"]
-  },
-  "ipfs": { "mode": "bundled" },
-  "logger": { "log-level": 2, "log-to-files": false },
-  "metrics": { "metrics-exporter-enabled": false, "metrics-port": 9090 },
-  "network": { "name": "inmemory" },
-  "node": {},
-  "state-store": { "mode": "fs", "local-directory": "~/.ceramic/statestore/" },
-  "indexing": {
-    "db": "sqlite:./ceramic-indexing/ceramic.sqlite",
-    "allow-queries-before-historical-sync": true,
-    "models": [
-      "kjzl6hvfrbw6c57p6t99krxnhqbxb24suuvxfhq1hnqujfecq0fap356ai7sjk9",
-      "kjzl6hvfrbw6c9m9brthgvymjbe0wzk8svf9i4trl7a0cm87e4hi3zyimpjxexf",
-      "kjzl6hvfrbw6c7gxr84vvivh8c5awz1eycoj793oixk68zw7k3z3zk5uh06fwxl",
-      "kjzl6hvfrbw6c57p6t99krxnhqbxb24suuvxfhq1hnqujfecq0fap356ai7sjk9"
-    ]
-  }
-}
+{"anchor":{},"http-api":{"cors-allowed-origins":[".*"],"admin-dids":["did:key:z6MkuPBmNwPhefW1Nezj5BioRi4wPh8jXJLSjesPcCDotozM"]},"ipfs":{"mode":"bundled"},"logger":{"log-level":2,"log-to-files":false},"metrics":{"metrics-exporter-enabled":false,"metrics-port":9090},"network":{"name":"testnet-clay"},"node":{},"state-store":{"mode":"fs","local-directory":"~/.ceramic/statestore/"},"indexing":{"db":"sqlite:///Users/markkrasner/.ceramic/indexing.sqlite","allow-queries-before-historical-sync":true,"models":[]}}
```

Comment:
This looks good, however, I could suggest against hardcoding the specific user name for a sqlite db path. I am specifically speaking about this section: 

"indexing":{"db":"sqlite:///Users/markkrasner/.ceramic/indexing.sqlite"

Also, can we make sure the contents in this file is formatted so that it's easier to read? Not sure if that just Github messing things up, but the config file seems to be all in one line.